pipelines/pyproject.toml

```diff
@@ -1,31 +1,33 @@
-[project]
-name = "turbo-template"
-authors = [
-    {name = "Example User", email = "user@example.com"},
-]
-description = "Turbo Template"
+[tool.poetry]
+name = "pipelines"
+version = "0.1.0"
+authors = ["Example User <user@example.com>"]
+description = "Vertex AI Pipelines end-to-end sample"
 readme = "README.md"
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Developers",
     "Programming Language :: Python :: 3.7",
 ]
-requires-python = ">=3.7"
-dynamic = ["version"]
-dependencies = [
-    "kfp == 1.8.21",
+packages = [
+    { include = "pipelines", from = "src" },
 ]
 
-[project.optional-dependencies]
-tests = [
-    "google-cloud-bigquery == 2.30.0",
-    "pytest >= 7.3.1,<8.0.0",
-    "coverage = ==7.2.5"
-]
+[tool.poetry.dependencies]
+python = ">=3.7,<3.11"
+Jinja2 = ">=3.0.1,<4.0.0"
+google-cloud-aiplatform = "1.24.1"
+google-cloud-pipeline-components = "1.0.42"
+bigquery-components = { path = "../components/bigquery-components", develop = true }
+vertex-components = { path = "../components/vertex-components", develop = true }
+
+[tool.poetry.group.dev.dependencies]
+pytest = ">=7.3.1,<8.0.0"
+pre-commit = ">=2.14.1,<3.0.0"
```

Comment:
Missing `"coverage = ==7.2.5"` package

Reply:
I'll add using a hotfix, the PR has been merged.

Reply:
874d282

Reply:
Done